portfolio/src/main/webapp/request.js

```diff
@@ -25,4 +25,105 @@ class ScheduleRequest {
   }
 }
 
-// TODO(tomasalvarez): write function to fetch results and display results
+// Once a call is made to the server for scheduling this array will be a local
+// copy of the results.
+var scheduledTasks = [];
```

Comment:
Is this local variable necessary? I think you render the scheduled tasks on the DOM, right? So you may not need this local variable. If we have a variable like this then it can be burdensome to keep it synced with the DOM.

Reply:
Currently this gets updated when you hit "Start Scheduling". I'd like to make the scheduled task card include all of the information (not just name and time) and then I will remove this local variable and get the information from the dom. That will probably happen in a PR soon.